go-controller/pkg/ovn/port_cache.go

```diff
@@ -0,0 +1,94 @@
+package ovn
+
+import (
+	"fmt"
+	"net"
+	"sync"
+	"time"
+
+	"k8s.io/klog"
+)
+
+type portCache struct {
+	sync.RWMutex
+	stopChan <-chan struct{}
+	cache    map[string]*lpInfo
+}
+
+type lpInfo struct {
+	name          string
+	uuid          string
+	logicalSwitch string
+	ip            net.IP
+	mac           net.HardwareAddr
+	// expires, if non-nil, indicates that this object is scheduled to be
+	// removed at the given time
+	expires time.Time
+}
+
+func newPortCache(stopChan <-chan struct{}) *portCache {
+	return &portCache{
+		stopChan: stopChan,
+		cache:    make(map[string]*lpInfo),
+	}
+}
+
+func (c *portCache) get(logicalPort string) (*lpInfo, error) {
+	c.RLock()
+	defer c.RUnlock()
+	if info, ok := c.cache[logicalPort]; ok {
+		klog.V(5).Infof("port-cache(%s): port found in cache", logicalPort)
```

Comment:
@dcbw L40 could be noisy as well and the log file would just consist of this when you have 1000s of pods in the cluster. Either we log this is at V(6) or delete the whole log message.

Reply:
@girishmg done